saleor/graphql/attribute/types.py

```diff
@@ -410,14 +415,18 @@ class AttributeValueSelectableTypeInput(BaseInputObjectType):
 
     class Meta:
         description = (
-            "Represents attribute value. If no ID provided, value will be resolved. "
-            + ADDED_IN_39
+            "Represents attribute value. If no ID or external reference provided, "
+            "value will be resolved. If externalReference and value is provided then "
+            "new attribute value will be created." + ADDED_IN_39
```

Comment:
IMO this description should be more detailed, are there seem to be three different ways it can work depending on the combination of arguments.